website/pages/docs/secrets/openldap/index.mdx

```diff
@@ -61,6 +61,34 @@ This plugin currently supports LDAP v3.
     $ vault read openldap/static-role/hashicorp
     ```
 
+## LDAP Password Policy
+
+The OpenLDAP secret engine does not hash or encrypt passwords prior to modifying 
+values in LDAP. This behavior can cause plaintext passwords to be stored in LDAP.
+
+To avoid having plaintext passwords stored an LDAP password policy (ppolicy) should 
```

Comment:
Can you add a blurb explicitly stating this is referring to the LDAP password policies and not the Vault password policies? Maybe something like "... not to be confused with Vault password policies..."

Reply:
Done.